udata/auth/views.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+from flask import Blueprint, current_app, request, redirect
+from udata.auth import current_user
+
+auth = Blueprint('auth', __name__)
+
+@auth.before_app_request
+def ensure_https_authenticated_users():
+    # Force authenticated users to use https
+    if (not current_app.config.get('TESTING', False)
+            and current_app.config.get('USE_SSL', False)
+            and current_user.is_authenticated()
+            and not request.is_secure):
+        return redirect(request.url.replace('http://', 'https://'))
```

Comment:
I'm surprised we're doing it at the app level given that to me it's a bakcend/server job. @noirbizarre did you do that for double-safety?